From 59e7a68bafebf4d45034538b1910c7660dc2cca5 Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Thu, 17 Mar 2016 16:24:32 +0900
Subject: [PATCH 587/909] arm64: dts: r8a7795: Add the emergency node in
 thermal-zones

commit ebec63e6de127d0b47ffd193e637754c4b8058bf from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

[dien.pham.ry: arm64: dts: r8a7795: Add CA53 for EMS target CPU]

This patch adds emergency shutdown support for R8A7795 SoC.

[gaku.inami.xw: disable emergency node to avoid thermal zone warning]
Signed-off-by: Hien Dang <hien.dang.eb@renesas.com>
Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 2c29964c67a7..20a32ab4ddf9 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -3369,6 +3369,20 @@
 	};
 
 	thermal-zones {
+		emergency {
+			polling-delay = <1000>;
+			on-temperature = <110000>;
+			off-temperature = <95000>;
+			target_cpus = <&a57_1>,
+				      <&a57_2>,
+				      <&a57_3>,
+				      <&a53_0>,
+				      <&a53_1>,
+				      <&a53_2>,
+				      <&a53_3>;
+			status = "disabled";
+		};
+
 		sensor_thermal1: sensor-thermal1 {
 			polling-delay-passive = <250>;
 			polling-delay = <0>;
-- 
2.17.1

